### ISSUE TITLE
Error while building docker

### ISSUE THREAD

**Issue by project-autosi** [BMW-InnovationLab/BMW-TensorFlow-Training-GUI#2]
Step 7/12 : RUN $(npm bin)/ng build --prod --build-optimizer
 ---> Running in 8e8f5cdb6770
Browserslist: caniuse-lite is outdated. Please run next command `npm update`

Date: 2019-12-20T12:44:35.601Z
Hash: c33a9b977eea03abeb23
Time: 12580ms
chunk {0} runtime-es5.741402d1d47331ce975c.js (runtime) 1.41 kB [entry] [rendered]
chunk {1} main-es5.4af9b61479361f268d39.js (main) 128 bytes [initial] [rendered]
chunk {2} polyfills-es5.6d628ad86fb70cb4472e.js (polyfills) 68.1 kB [initial] [rendered]

ERROR in node_modules/codelyzer/angular/styles/cssAst.d.ts(38,9): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/codelyzer/angular/styles/cssAst.d.ts(39,9): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/codelyzer/util/function.d.ts(21,9): error TS1086: An accessor cannot be declared in an ambient context.
node_modules/codelyzer/util/function.d.ts(22,9): error TS1086: An accessor cannot be declared in an ambient context.

ERROR: Service 'user_interface' failed to build: The command '/bin/sh -c $(npm bin)/ng build --prod --build-optimizer' returned a non-zero code: 1

Reffered -
https://github.com/postcss/autoprefixer/issues/1184
https://github.com/Cezerin2/cezerin2/issues/55
https://stackoverflow.com/questions/55271798/browserslist-caniuse-lite-is-outdated-please-run-next-command-npm-update-cani
but couldn't resolve the issue. Any suggestions and help will be really appreciated.
thanks in advance.

**Comment by JoeSL** [comment]
The error has to do with the "codelyzer" version. Please provide a picture of the package.json file found in your gui folder.

**Comment by Marc-Kamradt** [comment]
The problem was fixed with the update "codelyzer version change to prevent TS1086" from 12/19/2019

**Comment by project-autosi** [comment]
Thankyou @Marc-Kamradt  I was able to build successfully. But on http://0.0.0.0:4200/training I got this error 
![bmw_il](https://user-images.githubusercontent.com/59088985/71611988-2fdf4580-2bc3-11ea-9702-f88c32105b6e.png)

API down or unknown error. Although I was able to open test with swagger BMW InnovationLab tensorflow cpu inference Automation on port 4343.
What could be the issue. Thanks in advance.
